src/electionguard/encode.py

```diff
@@ -0,0 +1,50 @@
+"""Converting module that define padding and default settings for byte order and byte encoding."""
+
+from enum import IntEnum
+
+
+BYTE_ORDER = "big"
+BYTE_ENCODING = "utf-8"
+
+PAD_INDICATOR_SIZE = 2
+_PAD_BYTE = b"\x00"
+
+
+class PaddedDataSize(IntEnum):
+    """Define the sizes for padded data."""
+
+    Bytes_32 = 32 - PAD_INDICATOR_SIZE
+    Bytes_64 = 64 - PAD_INDICATOR_SIZE
+    Bytes_128 = 128 - PAD_INDICATOR_SIZE
+    Bytes_256 = 256 - PAD_INDICATOR_SIZE
+    Bytes_512 = 512 - PAD_INDICATOR_SIZE
+
+
+class TruncationError(ValueError):
+    """A specific truncation error to indicate when padded data is truncated."""
+
+
+def add_padding(
+    message: bytes, size: PaddedDataSize, allow_truncation: bool = True
```

Comment:
I defaulted to allow truncation as the group had decided but this setting could be added to config quite quickly. Valid suggestion.

Reply:
`allow_write_in_truncation` for example